app/cronjob/trailingTradeHelper/queue.js

```diff
@@ -1,76 +1,104 @@
-const config = require('config');
-const Queue = require('bull');
```

Comment:
Hmm, this is not a queue helper anymore. 
I feel like it's more like a `processor`. Or we can think of a better name for this.

You don't need to change now. I think we can refactor it later on.

Reply:
Let me refactor the code a bit.

Reply:
@uhliksk 

I've refactored your queue code to use the chain of responsibilities pattern. 
It should make it easier to read and follow the code.

https://github.com/chrisleekr/binance-trading-bot/pull/562/commits/9e7ec0998df555bc5920326b4c790d7b7bbdfcf6

If you have any feedback, please let me know. :+1: